packages/ipfs-http-client/src/files/rm.js

```diff
@@ -20,7 +21,15 @@ export const createRm = configure(api => {
       headers: options.headers
     })
 
-    await res.text()
+    const body = await res.text()
+    // we don't expect text body to be ever present
+    // (if so, it means an error such as https://github.com/ipfs/go-ipfs/issues/8606)
+    if (body !== '') {
+      /** @type {Error} */
+      const error = new HTTP.HTTPError(res)
+      error.message = body
+      throw error
+    }
```

Comment:
@achingbrain would this work around be acceptable?

Reply:
:groan: